experiments/jz/joint_training_toy/joint_training_toy1/03_create_dataset.slurm

```diff
@@ -44,14 +44,12 @@ REPO_DIR="${DATASETS_CUSTOM}/${myarray[-1]}"
 
 # Now we launch the script that will perform the preprocessing of the dataset
 # Feel free to add any arguments you like (change me!)
-python bsmetadata/train.py \ # (change me and remove the comment! if you have a specific script)
+python bsmetadata/train.py \
     data_config.experiment="with_metadata" \
-    data_config.metadata_config.metadata_list=["html"] \ # (change me and remove the comment!)
+    data_config.metadata_config.metadata_list=["entity", "timestamp", "url", "website_description"] \
     data_config.metadata_config.max_seq_len=1024 \
     data_config.dataset_name="${REPO_DIR}" \
-    data_config.train_file="XXX" \ # (change me and remove the comment! e.g "nq-train-0\[0-2\].jsonl.gz" or remove arg)
```

Comment:
Still not 100% sure, but I think that the automatic format detection feature need to know the file to include (like above)